lisa/sut_orchestrator/azure/platform_.py

```diff
@@ -246,6 +246,8 @@ class AzurePlatformSchema:
 
     shared_resource_group_name: str = AZURE_SHARED_RG_NAME
     resource_group_name: str = field(default="")
+    # specify resource group location
+    resource_group_location: Optional[str] = field(default=None)
```

Comment:
add one for shared_resource_group_location, and set its default value as `shared_resource_group_name`

Reply:
fixed